### PR TITLE
Fix golangci-lint (part 1)

### PR DESCRIPTION
**Pull request by osinstom**
This is the first from series of PRs to fix golangci-lint. 

I propose to relax Golangci-lint requirements as they were really strict. My proposal is to use the currently configured linters. We might want to add more linters in the future.

### CODE REVIEW

**Comment by krsna1729** [comment]
The downside is we do not get the annotations in the PR

**Comment by krsna1729** [comment]
Can we support both, i.e., use golangci-lint action instead of make target in github. Locally they can still use the make target.

**Comment by krsna1729** [comment]
if we can reduce the errors < 10 (limit on annotations) they will be visible on the PR itself.

File: pfcpiface/messages-session.go [hunk 1]
```diff
@@ -19,8 +19,12 @@ const (
 	DefaultDDNTimeout = 20
 )
 
-var errAssocNotFound = errors.New("no association found for NodeID")
-var errAllocateSession = errors.New("unable to allocate new PFCP session")
+// errors
+var (
+	writeToFastpathErr = errors.New("write to FastPath failed")
+	assocNotFoundErr   = errors.New("no association found for NodeID")
+	allocateSessionErr = errors.New("unable to allocate new PFCP session")
```
Comment (krsna1729) [inline pfcpiface/messages-session.go:26]: image

err<>

File: pfcpiface/messages.go [hunk 2]
```diff
@@ -98,12 +98,17 @@ func (pConn *PFCPConn) HandlePFCPMsg(buf []byte) {
 	case message.MsgTypeSessionDeletionRequest:
 		reply, err = pConn.handleSessionDeletionRequest(msg)
 	case message.MsgTypeSessionReportResponse:
-		pConn.handleSessionReportResponse(msg)
+		{
+			_, err := pConn.handleSessionReportResponse(msg)
+			if err != nil {
+				log.WithField("error", err).Warn("Failed to handle PFCP session report response, ignoring..")
+			}
+		}
```
Comment (krsna1729) [inline pfcpiface/messages.go:106]: The error check at the end does not service this one?

**Comment by osinstom** [comment]
@krsna1729 thanks for review. I agree it's good to have both, the problem that I saw was a diverged linter configuration (one in Github workflow file and another in the `.golangci.yml`. However, I fixed it and both CI jobs and local `make golint` use the same linter configuration.

**Review by krsna1729 — APPROVED** [review]

**Pull request merged**